emily/handler/src/api/handlers/withdrawal.rs

```diff
@@ -335,12 +335,12 @@ pub async fn update_withdrawals_signer(
         api_state.error_if_reorganizing()?;
 
         // Signers are only allowed to update withdrawals to the accepted state.
-        let is_unauthorized = body
+        let is_allowed = !body
```

Comment:
I think there was another instance down below in another function as well

Reply:
Whoops, [eb3fe2b](https://github.com/stacks-sbtc/sbtc/pull/1586/commits/eb3fe2b2112f16525965084e0b31f8796b8dcf02) actually